vanilla/vanilla
# Fix width in popups with the richEditor mounted

Closes https://github.com/vanilla/support/issues/1257

**Steps to test:**
- You can have keystone or any other theme enabled, go to a discussion and click on "Flag" and then "Warn"
- A full width popup will display, with the richEditor mounted.
- Checkout this branch and repeat the steps above, the popup should have a max of 600px on large screens and full width on mobile.
